homeassistant/components/sensor/viaggiatreno.py

```diff
@@ -0,0 +1,137 @@
+"""
+Trenitalia sensor.
+
+This simple sensor uses ViaggiaTreno APIs to provide information about Italian public railroads
+status.
+"""
+import datetime
+import logging
+import urllib.request
+import json
+import re
+import voluptuous as vol
+from urllib.parse import quote, urljoin
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = []
+
+log = logging.getLogger(__name__)
```

Comment:
To be aligned with the existing platform, please use `_LOGGER = logging.getLogger(__name__)`. If needed, would it be much fast to replace something with `sed`.